src/mina-signer/src/signature.ts

```diff
@@ -44,7 +44,7 @@ export {
 };
 
 const networkIdMainnet = 0x01n;
-const networkIdTestnet = 0x00n;
```

Comment:
This is where constants used during signing are set, changing the networkId to devnet will not change the underlying constants so the signatures themselves will not be changed